### ISSUE TITLE
Manual Input

### ISSUE THREAD

**Issue by RobinDev03** [statping-ng/statping-ng#200]
Hey,

is it possible, to make an "Check-Type" where I can manage the state manually?
I have some services, that cannot be monitored automatically (Cooling in the Datacenter etc...) How can I show these?

**Comment by bretdavi89** [comment]
When creating a service, you can set the service type to "Static Service", which you can then manually update. I'm unaware of a way to modify the status directly via the UI, but you can update the status via the API: https://github.com/statping-ng/statping-ng/wiki/API

Look for "Update Static Service" in the docs, and it's a simple PATCH call to change the status of the service. Been working on setting up statping myself, so hope this was helpful

**Comment by RobinDev03** [comment]
Hey,

thanks.
A function in the UI would be great for that.

**Comment by jemand771** [comment]
yeah, static services should be the way to go for this. I'm thinking about just a set of buttons in the UI to manually fire a checkin, although I'm not sure if a checkin can result in a negative state

either way this will probably have to be addressed after #178
